config/locales/en.yml

```diff
@@ -1472,12 +1472,15 @@ en:
         total_capital:
           text: Total capital
           hint: ""
+          alt_hint: ""
         pensioner_capital_disregard:
           text: Pensioner disregard
           hint: Applied to total capital and capped at £100,000
+          alt_hint: Applied to total capital up to a maximum of £100,000
         smod_disregard:
           text: Disputed asset disregard
           hint: Equal to the assessed value of all assets marked as disputed and capped at £100,000
+          alt_hint: Applied to remaining capital after disputed asset disregard has been applied, and up to a maximum of £100,000
```

Comment:
This is in the wrong place. We want to show this text on the _pensioner_ disregard line _if_ there is non-zero SMOD.